guide/english/certifications/javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords/index.md

```diff
@@ -38,21 +36,14 @@ We need to change `var` to `let` in our function scope and add `let` to our bloc
     return i;
     }
 ```
-![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://codepen.io/dylantyates/pen/wxwxRd' target='_blank' rel='nofollow'>Run Code</a>
-
-# Code Explanation:
-
-By using `let` you can declare variables in relation to their scope.
-
-#### Relevant Links
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") [Run code at codepen.io](https://codepen.io/dylantyates/pen/wxwxRd)
```

Comment:
```suggestion

```

Since we are in the process of phasing out links containing user controlled code, I'd appreciate if you remove this line